chapter_multilayer-perceptrons/kaggle-house-price.md

```diff
@@ -430,6 +453,26 @@ def train(net, train_features, train_labels, test_features, test_labels,
     return train_ls, test_ls
 ```
 
+```{.python .input}
+#@tab paddle
+def train(net, train_features, train_labels, test_features, test_labels,
+          num_epochs, learning_rate, weight_decay, batch_size):
+    train_ls, test_ls = [], []
+    train_iter = d2l.load_array((train_features, train_labels), batch_size)
+    # 这里使用的是Adam优化算法
+    optimizer = paddle.optimizer.Adam(learning_rate=learning_rate*1.0, parameters=net.parameters(),weight_decay = weight_decay*1.0)
```

Comment:
space after net.parameters(),
remove space around "=", 
why learning_rate * 1.0, instead of only learning_rate?

Reply:
The learning_rate should be float, otherwise it will raise an exception

Reply:
It seems we do not have this problem in other DL libs. Is it mandatory in paddlepadle?